Makefile

```diff
@@ -161,39 +155,9 @@ fmt:
 	@$(GOFMT) -e -s -l -w $(ALL_SRC)
 	./scripts/updateLicenses.sh
 
-.PHONY: lint-gosec
-lint-gosec:
-	time gosec -quiet -exclude=G104,G107 ./...
-
-.PHONY: lint-staticcheck
-lint-staticcheck:
-	@cat /dev/null > $(LINT_LOG)
-	time staticcheck ./... \
-		| grep -v \
-			-e model/model.pb.go \
-			-e proto-gen \
-			-e _test.pb.go \
-			-e thrift-gen/ \
-			-e swagger-gen/ \
-		>> $(LINT_LOG) || true
-	@[ ! -s "$(LINT_LOG)" ] || (echo "Detected staticcheck failures:" | cat - $(LINT_LOG) && false)
-
 .PHONY: lint
-lint: lint-staticcheck lint-gosec
-	$(GOVET) ./...
-	$(MAKE) go-lint
-	@echo Running go fmt on ALL_SRC ...
-	@$(GOFMT) -e -s -l $(ALL_SRC) > $(FMT_LOG)
-	./scripts/updateLicenses.sh >> $(FMT_LOG)
```

Comment:
This must remain

Reply:
Nice catch, I've added this back into the `lint` target. Tested by removing the license from `grpc_handler.go`:
```
$ make lint
golangci-lint run
./scripts/updateLicenses.sh > .fmt.log
License check failures, run 'make fmt'
cmd/collector/app/handler/grpc_handler.go
make: *** [lint] Error 1
```